metadata/fisheye.xml

```diff
@@ -7,12 +7,12 @@
 		<option name="toggle" type="activator">
 			<_short>Toggle</_short>
 			<_long>Toggles fisheye with the specified key.</_long>
-			<default>&lt;super&gt; KEY_F</default>
+			<default>&lt;super&gt; &lt;ctrl&gt; BTN_LEFT</default>
```

Comment:
I would rather see this combined with super+F than replace it.

Reply:
It is already used for the grid.
Rationale for a button is it affects the cursor.
I rather prefer the fancy effects be on the <kbd>Control</kbd> in general.

Reply:
maybe ctrl-super-F? I don't think this should be a button binding by default.